cli/commands/deploy.rs

```diff
@@ -102,7 +102,15 @@ impl Deploy {
             match request.send(&endpoint) {
                 Ok(_) => println!("✅ Successfully deployed '{}' to the local development node.", program.id()),
                 Err(error) => {
-                    bail!("❌ Failed to deploy '{}' to the local development node: {}", program.id(), error);
+                    if let Ok(ureq::Error::Status(code, response)) = error.downcast::<ureq::Error>() {
```

Comment:
@HarukaMa taught me that you could downcast the error and get the response as a string.

His original code was:

```
            // Send the deployment request to the local development node.
            match request.send(&endpoint) {
                Ok(_) => println!("✅ Successfully deployed '{}' to the local development node.", program.id()),
                Err(error) => {
                    match error.downcast::<ureq::Error>() {
                        Ok(ureq::Error::Status(code, response)) => {
                            bail!("❌ Failed to deploy '{}' to the local development node: {} {:?}", program.id(), code, response.into_string());
                        }
                        _ => {}
                    }
                    bail!("❌ Failed to deploy '{}' to the local development node", program.id());
                }
            };
```
I refactored the above slightly to use `if let`.  @HarukaMa explained to me, his match statement was in an attempt to ensure the error still gets passed if the `downcast` expression is not matched.